packages/pxp/pxp.1.2.9/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+name: "pxp"
+version: "1.2.9"
+maintainer: "codinuum@me.com"
+authors: "Gerd Stolpmann <gerd@gerd-stolpmann.de>"
+homepage: "http://projects.camlcity.org/projects/pxp.html"
+dev-repo: "git+https://gitlab.camlcity.org/gerd/lib-pxp.git"
+bug-reports: "ocaml-pxp-users@orcaware.com"
+license: "MIT/X11"
+patches: "408.patch" {ocaml:version >= "4.08.0"}
+build: [
+  [
+    "./configure"
+    "-without-wlex"
+    "-without-wlex-compat"
+    "-lexlist"
+    "utf8,iso88591"
+  ]
+  [make "all"]
+  [make "opt"]
+]
+install: [make "install" "MANDIR=%{man}%"]
+remove: [
+  ["ocamlfind" "remove" "pxp-engine"]
+  ["ocamlfind" "remove" "pxp-lex-iso88591"]
+  ["ocamlfind" "remove" "pxp-lex-utf8"]
+  ["ocamlfind" "remove" "pxp-pp"]
+  ["ocamlfind" "remove" "pxp-ulex-utf8"]
+  ["ocamlfind" "remove" "pxp"]
+]
+depends: [
+  "ocaml" {>= "4.01"}
+  "ocamlfind"
```

Comment:
```suggestion
  "ocamlfind" {build}
```
Are you sure ocamlfind/Findlib is actually used outside at runtime?

Reply:
I'm not sure... but `INSTALL` has the following sentence:
> The installation procedure defined in the Makefile requires findlib [2] to work [3].

Reply:
if it is only used to install and build then it is safe to tag it as a build dependency

Reply:
The only use of the `Findlib` library I could find is in `rtests/Makefile`